### ISSUE TITLE
bro why is the download size so big

### ISSUE THREAD

**Issue by H4ppy-04** [vanRooijen/dot_files#4]
![image](https://github.com/vanRooijen/dot_files/assets/101031214/e351459b-db0c-4df5-a632-c02844cb67fa)

**Comment by vanRooijen** [comment]
![huge_file_size](https://github.com/vanRooijen/dot_files/assets/116570950/b4200b3e-7434-47be-937d-6ecbcb2d96ec)

The wallpapers are really big files, like frighteningly big.
They are made by running them through an algorithm to make the colors more dull effectively to be easier on the eye, as well as having general grains added on GIMP.
Both of which are not highly optimized in terms of  compression. Like they didn't even try.
I could have compressed it but, I guess I didn't feel like it. (Negligible amount of space for me.)

**Comment by H4ppy-04** [comment]
I helped develop one of those algorithms (gruvbox factory). But yea, 
that makes sense, I can understand why compression would not be a 
priority.

**Comment by vanRooijen** [comment]
First of all that's really cool, I have used gruvbox factory before for some more cartoony projects, however given the realistic images I used lutgen this time. 
I'm sat here thinking now, it will probably be 99% the grain effect as this will create a significant loss of flat color areas, thus making it harder for compression algorithms (especially lossless ones like those typically used with PNG files) to efficiently reduce file size without losing detail.
So basically I though the easiest thing to do would be to convert to jpg as their compression is more aggressive and importantly not lossless.
So I did. Anyway it still totals at probably about 50 Mb for all of them together, but that is already a lot better than 500 Mb.
(Will look into this issue more at a later point probably, for now I just pushed the 50 Mb.)
